### PR TITLE
fedora-40 image refresh and naughty pattern for 7139

### PR DESCRIPTION
**Pull request by mvollmer**
* [x] image-refresh fedora-40

 - [x] https://github.com/cockpit-project/cockpit/pull/21321 and land https://bodhi.fedoraproject.org/updates/FEDORA-2024-6c751cfc0a ; or land https://github.com/candlepin/subscription-manager-cockpit/pull/84
 - [x] https://github.com/cockpit-project/cockpit-files/pull/841 in lock step
 - [x] https://github.com/cockpit-project/cockpit/pull/21322 in lock step
 - [x] https://github.com/cockpit-project/cockpit-machines/pull/1914
 - [x] https://github.com/cockpit-project/cockpit-machines/pull/1925

Fixes https://github.com/cockpit-project/cockpit/issues/21301

### CODE REVIEW

**Comment by mvollmer** [comment]
This should also trigger in fedora-40 already. Let's make an image right away...

**Comment by cockpituous** [comment]
image-refresh fedora-40 done: https://github.com/cockpit-project/bots/commits/image-refresh-fedora-40-20241125-101752

**Comment by cockpituous** [comment]
Success. Log: https://cockpit-logs.us-east-1.linodeobjects.com/image-refresh-fedora-40-1d76dfbd-20241125-095709/log.html

File: naughty/fedora-40/7139-tracer-segfaults-in-the-past [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# testBasic (__main__.TestHistoryMetrics.testBasic)
```
Comment (martinpitt) [inline naughty/fedora-40/7139-tracer-segfaults-in-the-past:1]: Please copy that to fedora-41 as well, see #7123

Comment (martinpitt) [inline naughty/fedora-40/7139-tracer-segfaults-in-the-past:1]: And it also affects TestMultiCPU.testCPUUsage

**Review by martinpitt — CHANGES_REQUESTED** [review]

**Comment by martinpitt** [comment]
(Copied from the parallel image refresh #7126 -- there is no reason to believe that this refresh magically fixes any of these):

Phew, this is a big one. There's the python crashes from #21301, [files needs a pixel update](https://cockpit-logs.us-east-1.linodeobjects.com/pull-7126-31b95e35-20241123-230312-fedora-40-cockpit-project-cockpit-files/log.html) due to the new cockpit version enabling "open in terminal", sub-man causes an oops in the same manner as in #7123, and composer failed on an internal npm mirror failure (retried that one).

**Comment by mvollmer** [comment]
> (Copied from the parallel image refresh #7126 -- there is no reason to believe that this refresh magically fixes any of these):

Oops, I wasn't aware yet of that PR... sorry.

**Comment by mvollmer** [comment]
> [TestMultiCPU.testCPUUsage](https://cockpit-logs.us-east-1.linodeobjects.com/pull-7140-7b80df96-20241125-111406-fedora-40-other-cockpit-project-cockpit/log.html#39) not caught by naughty?

My mistake. Man, I am bad at multi-tasking...

**Comment by martinpitt** [comment]
`sudo dnf info cockpit-bridge` shows 329.1 now, so refreshing.

**Comment by cockpituous** [comment]
Success. Log: https://cockpit-logs.us-east-1.linodeobjects.com/image-refresh-fedora-40-fdac239d-20241126-122414/log.html

**Comment by martinpitt** [comment]
https://github.com/cockpit-project/cockpit-machines/pull/1925 landed, so retrying the machines tests. cockpit and -files only fail the pixel tests. After machines goes green, I'll land the pixel tests and then do a final rebase to land this fully green -- otherwise we'll ruin the flake statistics. It's evening and fairly quiet, so I think we can afford to do it this way.

**Review by martinpitt — APPROVED** [review]
:rocket: *exhale* 

Thanks @mvollmer ! Great teamwork!

**Pull request merged**